demos/button/default.html

```diff
@@ -4,31 +4,34 @@
 	<meta charset="utf-8">
 	<title>jQuery UI Button - Default functionality</title>
 	<link rel="stylesheet" href="../../themes/base/all.css">
-	<script src="../../external/jquery/jquery.js"></script>
-	<script src="../../ui/core.js"></script>
-	<script src="../../ui/widget.js"></script>
-	<script src="../../ui/button.js"></script>
 	<link rel="stylesheet" href="../demos.css">
-	<script>
-	$(function() {
-		$( "input[type=submit], a, button" )
-			.button()
-			.on( "click", function( event ) {
-				event.preventDefault();
-			});
-	});
+	<script src="../../external/requirejs/require.js"></script>
+	<script src="../bootstrap.js">
+		$( ".widget input[type=submit], .widget a, .widget button" ).button();
+		$( "button, input, a" ).click( function( event ) {
+			event.preventDefault();
+		} );
```

Comment:
Ignore the chaining comment, I see why you did that now that I've looked at the full diff.